src/Raven.Server/Documents/Handlers/RetiredAttachmentHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="DocumentHandler.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System.Threading.Tasks;
+using Raven.Server.Documents.Handlers.Processors.Attachments.Retired;
+using Raven.Server.Routing;
+
+namespace Raven.Server.Documents.Handlers
+{
+    public sealed class RetiredAttachmentHandler : DatabaseRequestHandler
+    {
+        [RavenAction("/databases/*/attachments/retire/config", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
```

Comment:
the issue here is that this configuration contains credentials, IMO it should be DatabaseAdmin